doc/classes/EditorInspector.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorInspector" inherits="ScrollContainer" version="4.0">
 	<brief_description>
+		The tab located by default in the right hand side of the editor. It is used to edit properties of the node selected.
 	</brief_description>
 	<description>
+		The inspector is a tab by default located on the right hand side of the godot editor. It is used to edit the properties of the selected node. For example, if you hand an Sprite node selected and head to the inspector. You can edit the transform of it and other properties.The inspector is essential in the game development work flow.
```

Comment:
"godot" should be "Godot" (capitalize it). "an Sprite" should be "a Sprite", but this sentence is confusing and needs work. Missing space after "properties." "work flow" -> "workflow".

Reply:
Thanks for pointing that out, I just fixed it.